src/Numerics/Distributions/Logistic.cs

```diff
@@ -0,0 +1,514 @@
+            return new Logistic(mean, scale, randomSource);
+        }
+
+        /// <summary>
+        /// Constructs a logistic distribution from a mean and variance.
+        /// </summary>
+        /// <param name="mean">The mean (μ) of the logistic distribution.</param>
+        /// <param name="var">The variance (σ^2) of the logistic distribution. Range: (σ^2) > 0.</param>
+        /// <param name="randomSource">The random number generator which is used to draw random samples. Optional, can be null.</param>
+        /// <returns>A logistic distribution.</returns>
+        public static Logistic WithMeanVariance(double mean, double var, System.Random randomSource = null)
+        {
+            return WithMeanStdDev(mean, Math.Sqrt(var), randomSource);
+        }
+
+        /// <summary>
+        /// Constructs a logistic distribution from a mean and precision.
+        /// </summary>
+        /// <param name="mean">The mean (μ) of the logistic distribution.</param>
+        /// <param name="precision">The precision of the logistic distribution.  Range: precision > 0.</param>
+        /// <param name="randomSource">The random number generator which is used to draw random samples. Optional, can be null.</param>
+        /// <returns>A logistic distribution.</returns>
+        public static Logistic WithMeanPrecision(double mean, double precision, System.Random randomSource = null)
+        {
+            return WithMeanVariance(mean, 1 / precision, randomSource);
+        }
+
+        /// <summary>
+        /// A string representation of the distribution.
+        /// </summary>
+        /// <returns>a string representation of the distribution.</returns>
+        public override string ToString()
+        {
+            return $"Logistic(μ = {_mean}, s = {_scale})";
+        }
+
+        /// <summary>
+        /// Tests whether the provided values are valid parameters for this distribution.
+        /// </summary>
+        /// <param name="mean">The mean (μ) of the logistic distribution.</param>
+        /// <param name="scale">The scale (s) of the logistic distribution. Range: s > 0.</param>
+        public static bool IsValidParameterSet(double mean, double scale)
+        {
+            return scale > 0.0 && !double.IsNaN(mean);
+        }
+
+        /// <summary>
+        /// Gets the scale parameter of the Logistic distribution. Range: s > 0.
+        /// </summary>
+        public double Scale => _scale;
+
+        /// <summary>
+        /// Gets the mean (μ) of the logistic distribution.
+        /// </summary>
+        public double Mean => _mean;
+
+        /// <summary>
+        /// Gets the standard deviation (σ) of the logistic distribution. Range: σ > 0.
+        /// </summary>
+        public double StdDev => Math.Sqrt(Variance);
+
+        /// <summary>
+        /// Gets the variance of the logistic distribution.
+        /// </summary>
+        public double Variance => (Math.Pow(_scale, 2) * Math.Pow(Math.PI,2))/3;
```

Comment:
I think it's more efficient in this case to use __scale * _scale * Math.PI * Math.PI / 3;
or _scale * _scale * 3.289868133696452872944;